src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -136,6 +151,14 @@ def login(cmd, username=None, password=None, service_principal=None, tenant=None
         from azure.cli.core.auth.identity import ServicePrincipalAuth
         password = ServicePrincipalAuth.build_credential(password, client_assertion, use_cert_sn_issuer)
 
+
+    login_experience_v2 = cmd.cli_ctx.config.getboolean('core', 'login_experience_v2', fallback=True)
+    # Send login_experience_v2 config to telemetry
+    from azure.cli.core.telemetry import set_login_experience_v2
+    set_login_experience_v2(login_experience_v2)
+
+    select_subscription = interactive and sys.stdin.isatty() and sys.stdout.isatty() and login_experience_v2
```

Comment:
Should we also send `select_subscription` to telemetry? `login_experience_v2` stands for the config value, while `select_subscription` stands for whether interactive selection is actually invoked. @dcaro

Reply:
We'll consider adding telemetry for `select_subscription` in the future.